Node/playwright_vanilla/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I don't think this is required I don't think we can assume that the user will have vscode